app.js

```diff
@@ -0,0 +1,66 @@
+var app = {
+    Models: {},
+    Collections: {},
+    Views: {}
+};
+
+app.Models.TodoItem = Backbone.Model.extend({
+    defaults: {
+        // title: "",
+        completed: false
+    }
+});
+
+app.Collections.TodoItems = Backbone.Collection.extend({
+    model: app.Models.TodoItem,
+    // localStorage: new Store("backbone-todo")
+})
+
+app.Views.TodoApp = Backbone.View.extend({
+    el: ".todo",
+    initialize: function() {
+        this.newTodoField = this.$(".todo__new-todo-field");
+        this.markAllCompleteField = this.$(".mark-all-complete");
+        this.totalCount = this.$(".total-count");
+        this.remainingCount = this.$(".remaining-count");
+        this.completeCount = this.$(".complete-count");
+    }
+});
+app.Views.TodoItem = Backbone.View.extend({
+    tagName: "li",
+    className: "todo-item",
+    template: Handlebars.compile($("#todo-item-template").html()),
+    render: function() {
+        this.$el.html(this.template(this.model.toJSON()))
+        return this.$el;
+    }
+});
+app.Views.TodoItems = Backbone.View.extend({
+    el: ".todo-items",
+    initialize: function() {
+        // this.listenTo(this.collection, "add", );
+        // this.listenTo(this.collection, "remove", );
+
+        // app.todoItems.fetch();
+        this.render();
+    },
+    render: function() {
+        this.$el.html('');  // clear children
+        
+        let todoViews = this.collection.map(todoItem => {
```

Comment:
This will be const because it doesn't get modified.

Also, we will need to address this for memory leaks. If `render` is called a second time. All the existing views you've created here will be zombied. What you should do is keep reference to each of the views (usually on `this.childViews`.) Then you can iterate over them and call `remove()` which will clean them up. Then you create the fresh ones.

Reply:
Whoops, was in Swift mode (Swift has var and let for JS's var and const)